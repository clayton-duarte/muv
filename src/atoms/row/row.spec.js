import React from 'react';

import Row from '.';

it('should render a row', () => {
  const wrapper = mount(
    <Row />,
  );
  expect(wrapper).toMatchSnapshot();
});
