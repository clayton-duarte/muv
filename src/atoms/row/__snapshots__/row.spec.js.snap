// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render a row 1`] = `
<Row
  align="stretch"
  grow={1}
  justify="flex-start"
  theme={
    Object {
      "color": Object {
        "black": "#484848",
        "gray": "#f0f0f0",
        "overlay": "rgba(0, 0, 0, .15)",
        "primary": "#0082c8",
        "secondary": "#00baf2",
        "success": "#33bb33",
        "warn": "#dd3333",
        "white": "#fefefe",
      },
      "shape": Object {
        "base": "1rem",
        "border": ".125rem solid rgba(0, 0, 0, .15)",
        "margin": ".25rem .125rem",
        "padding": ".825rem 1.2rem",
        "radius": ".25rem",
        "shadow": "0 .25rem .25rem 0 rgba(0, 0, 0, .15)",
      },
      "text": Object {
        "color": "#484848",
        "size": "1rem",
      },
      "transition": Object {
        "time": ".2s ease",
      },
    }
  }
>
  <Styled(Container)
    align="stretch"
    grow={1}
    justify="flex-start"
    theme={
      Object {
        "color": Object {
          "black": "#484848",
          "gray": "#f0f0f0",
          "overlay": "rgba(0, 0, 0, .15)",
          "primary": "#0082c8",
          "secondary": "#00baf2",
          "success": "#33bb33",
          "warn": "#dd3333",
          "white": "#fefefe",
        },
        "shape": Object {
          "base": "1rem",
          "border": ".125rem solid rgba(0, 0, 0, .15)",
          "margin": ".25rem .125rem",
          "padding": ".825rem 1.2rem",
          "radius": ".25rem",
          "shadow": "0 .25rem .25rem 0 rgba(0, 0, 0, .15)",
        },
        "text": Object {
          "color": "#484848",
          "size": "1rem",
        },
        "transition": Object {
          "time": ".2s ease",
        },
      }
    }
  >
    <Container
      align="stretch"
      className="sc-bwzfXH jBHyaV"
      full={false}
      grow={1}
      height="auto"
      justify="flex-start"
      maxHeight="auto"
      maxWidth="auto"
      minHeight="auto"
      minWidth="auto"
      theme={
        Object {
          "color": Object {
            "black": "#484848",
            "gray": "#f0f0f0",
            "overlay": "rgba(0, 0, 0, .15)",
            "primary": "#0082c8",
            "secondary": "#00baf2",
            "success": "#33bb33",
            "warn": "#dd3333",
            "white": "#fefefe",
          },
          "shape": Object {
            "base": "1rem",
            "border": ".125rem solid rgba(0, 0, 0, .15)",
            "margin": ".25rem .125rem",
            "padding": ".825rem 1.2rem",
            "radius": ".25rem",
            "shadow": "0 .25rem .25rem 0 rgba(0, 0, 0, .15)",
          },
          "text": Object {
            "color": "#484848",
            "size": "1rem",
          },
          "transition": Object {
            "time": ".2s ease",
          },
        }
      }
      width="auto"
    >
      <styled.div
        align="stretch"
        className="sc-bwzfXH jBHyaV"
        full={false}
        grow={1}
        height="auto"
        justify="flex-start"
        maxHeight="auto"
        maxWidth="auto"
        minHeight="auto"
        minWidth="auto"
        theme={
          Object {
            "color": Object {
              "black": "#484848",
              "gray": "#f0f0f0",
              "overlay": "rgba(0, 0, 0, .15)",
              "primary": "#0082c8",
              "secondary": "#00baf2",
              "success": "#33bb33",
              "warn": "#dd3333",
              "white": "#fefefe",
            },
            "shape": Object {
              "base": "1rem",
              "border": ".125rem solid rgba(0, 0, 0, .15)",
              "margin": ".25rem .125rem",
              "padding": ".825rem 1.2rem",
              "radius": ".25rem",
              "shadow": "0 .25rem .25rem 0 rgba(0, 0, 0, .15)",
            },
            "text": Object {
              "color": "#484848",
              "size": "1rem",
            },
            "transition": Object {
              "time": ".2s ease",
            },
          }
        }
        width="auto"
      >
        <div
          className="sc-bwzfXH jBHyaV sc-bdVaJa eGlEDl"
          height="auto"
          width="auto"
        >
          row
        </div>
      </styled.div>
    </Container>
  </Styled(Container)>
</Row>
`;
